### ISSUE TITLE
rabbitmq-dotnet-client 6.1 crashes rebus at startup

### ISSUE THREAD

**Issue by mathiasnohall** [rebus-org/Rebus.RabbitMq#69]
using the new version of dotnet rabbitMQ client 6.1.0 seems to make rebus crash at startup in an MVC ASPNET Core 3.1 application.

```
Rebus.Injection.ResolutionException: 'Could not resolve Rebus.RabbitMq.RabbitMqTransport with decorator depth 0 - registrations: Rebus.Injection.Injectionist+Handler'

TypeLoadException: Could not load type 'RabbitMQ.Client.IQueueingBasicConsumer' from assembly 'RabbitMQ.Client, Version=6.0.0.0, Culture=neutral, PublicKeyToken=89e7d7c5feba84ce'.

```

using Rebus 6.3.0 and Rebus.RabbitMq 7.1.0

**Comment by mathiasnohall** [comment]
using 
Rebus version 6.3.0
Rebus.RabbitMq version 7.1.0

**Comment by mookid8000** [comment]
Sounds like Rebus.RabbitMq needs to have its RabbitMq.Client dependency updated.

**Comment by mookid8000** [comment]
OK, work is ongoing in making Rebus.RabbitMq work with version 6.1 of the driver.

Many of the tests are passing now, but some of them experience weird errors that indicates having received some kind of mangled message body.

Gonna park it for now – most recent code is on master.

**Comment by Ilya-by** [comment]
> Gonna park it for now

any news on updating the client driver version?

**Comment by mookid8000** [comment]
oh, it's been a long time since I've looked at it.... I'll see if I can get back to this in the coming week

**Comment by archimate** [comment]
waiting。。。

**Comment by wzychla** [comment]
Bumping this.

**Comment by mathiasnohall** [comment]
https://github.com/rebus-org/Rebus.RabbitMq/pull/71

solves this. I have tested it in our code enviroment and it works. Some unit tests in Rebus.RabbitMq.Tests are failing though

**Comment by mookid8000** [comment]
Ok cool! I'll have time to look at it on Friday

**Comment by mookid8000** [comment]
Should be fixed by #71 🙂
